docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "2"
+services:
+  app:
+    build: .
+    links:
+      - db
+    volumes:
+      - .:/app
+    environment:
+      - DATABASE_URL=postgres://appuser:test123@db/test_project
+      - DJANGO_SETTINGS_MODULE=config.settings_interactive
+      - PYTHONUNBUFFERED=yup
+    working_dir: /app/test_project
+    entrypoint: ["./wait-for-postgres.sh"]
+    ports:
+      - "${APP_PORT:-8000}:${APP_PORT:-8000}"
+    command: python manage.py runserver 0.0.0.0:${APP_PORT:-8000}
```

Comment:
Note that it wasn't necessary for me to set the _internal_ port to `APP_PORT` here, but I wanted to because the development server prints a message saying what port it's on, e.g.:

```
Starting development server at http://0.0.0.0:8000/
```

I wanted the port displayed in such messages to always be whatever the user's environment variable is set to, to avoid confusion, which is why I'm using `APP_PORT` inside the container too.

I used the same approach for the docs server below, too.